README.md

```diff
@@ -5,7 +5,60 @@ Analytics Top-N Plugin
 
 CDAP Plugin for getting top N rows sorted by a given field.
 
-* [Top-N Plugin](docs/TopN-batchaggregator.md)
+Use Case
+--------
+This plugin takes input record and keeps a given number rows with highest values in a given field. If the total number of rows is smaller than the given number, the output record will contain all rows sorted by their values in the given field in a descending order.
+
+Properties
+----------
+**topField:** Name of the field by which top results are sorted. It must be an existing field from the input schema with one of type ``int``, ``long``, ``float``, or ``double``.
+
+**topSize:** Maximum number of rows in the result. It must be a positive integer.
+
+**ignoreNull:** Whether to ignore records with null in the field by which records are sorted. Defaults to 'false' to treat null as smallest value.
+
+Example
+-------
+The plugin takes input record that have colums name and age. Then it output top 3 rows with largest age values without ignoring null values.
+
+{
+  "name": "TopN",
+  "type": "batchaggregator",
+  "properties": {
+     "topField": "age",
+     "topSize": "3",
+     "ignoreNull": "false"
+   }
+}
+
+For example, suppose the plugin receives the input record:
+
+    +================+
+    | name   |  age  |
+    +================+
+    | alice  |       |
+    | bob    |   1   |
+    | dave   |   6   |
+    +================+
+
+The output record will be:
```

Comment:
just realized this is the readme and not the doc. Please apply these changes to the doc.